rpm/kf5bluezqt-bluez5.spec

```diff
@@ -63,6 +71,13 @@ This package contains the development header files for kf5bluezqt
 %install
 %make_install
 
+doxygen ../doc/Doxyfile
+mkdir -p %{buildroot}/%{_docdir}/%{name}
+mkdir -p %{buildroot}/%{_docdir}/%{name}/search
+rm doc/html/installdox
+install -m 644 -D -t %{buildroot}/%{_docdir}/%{name} doc/html/* || true
+install -m 644 -D -t %{buildroot}/%{_docdir}/%{name}/search doc/html/search/*
```

Comment:
Maybe use plain `cp -r`? The permissions would likely be correct already on the source side anyway.

Reply:
The permissions indeed seem to be correct; changed to `cp -r`